tsl/test/expected/compression_bgw.out

```diff
@@ -205,3 +205,66 @@ ERROR:  permission denied to start compress_chunks background process as role "n
 \set ON_ERROR_STOP 1
 RESET ROLE;
 REVOKE NOLOGIN_ROLE FROM :ROLE_DEFAULT_PERM_USER;
+\c :TEST_DBNAME :ROLE_SUPERUSER
+DROP TABLE IF EXISTS conditions CASCADE;
+NOTICE:  drop cascades to table _timescaledb_internal.compress_hyper_2_4_chunk
+CREATE TABLE conditions(
+    time TIMESTAMPTZ NOT NULL,
+    device INTEGER,
+    temperature FLOAT
+);
+SELECT * FROM create_hypertable('conditions', 'time', 'device', 30,
+       chunk_time_interval => '1 day'::interval,
+       migrate_data => true);
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             7 | public      | conditions | t
+(1 row)
+
+INSERT INTO conditions
+SELECT time, (random()*30)::int, random()*80 - 40
+FROM generate_series(NOW() - INTERVAL '90 days', NOW(), '10 min') AS time;
+CREATE VIEW conditions_summary
+WITH (timescaledb.continuous,
+      timescaledb.refresh_interval='1 minute',
+      timescaledb.max_interval_per_job = '90 days') AS
+SELECT device,
+       time_bucket(INTERVAL '1 hour', "time") AS day,
+       AVG(temperature) AS avg_temperature,
+       MAX(temperature) AS max_temperature,
+       MIN(temperature) AS min_temperature
+FROM conditions
+GROUP BY device, time_bucket(INTERVAL '1 hour', "time");
+NOTICE:  adding index _materialized_hypertable_8_device_day_idx ON _timescaledb_internal._materialized_hypertable_8 USING BTREE(device, day)
+REFRESH MATERIALIZED VIEW conditions_summary;
```

Comment:
Shouldn't we run this with background works off for repeatability? (Also, maybe this test shouldn't be in this test file since it seems focused on bgw?)

Reply:
It will not make a difference if the background worker is running or not. The test was put in this file since that seemed to already have the pieces needed, no other reason.